src/battle/partner/twink_714CF0.c

```diff
@@ -1,5 +1,12 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "battle/partner/twink_714CF0", func_80238000_714CF0);
-
+ApiStatus func_80238000_714CF0(ScriptInstance *arg0) {
+    BattleStatus* battleStatus = BATTLE_STATUS;
+    if ((battleStatus->flags2 & 0x40) != 0) {
```

Comment:
Since this is a boolean (true or false), you can just do `if (battleStatus->flags2 & 0x40)`